resharper/src/resharper-unity/Psi/ShaderLab/Parsing/ShaderLab.psi

```diff
@@ -1225,16 +1345,27 @@ cullCommand
 
 cullValue
 :
-  FRONT_KEYWORD<SHADER_LAB_VALUE, Value>
-  | BACK_KEYWORD<SHADER_LAB_VALUE, Value>
-  | OFF_KEYWORD<SHADER_LAB_VALUE, Value>
-  | referencedProperty<SHADER_LAB_REFERENCE, ReferencedProperty>
+  falseLiteral{FALSE_LITERAL, FalseLiteral}
+  | cullOrientation<ORIENTATION, Orientation>
+;
+
+cullOrientation
+:
+  FRONT_KEYWORD<SHADER_LAB_KEYWORD, Keyword>
+  | BACK_KEYWORD<SHADER_LAB_KEYWORD, Keyword>
+  | variableReference<SHADER_LAB_REFERENCE, VariableReference>  // Reference to float, but which value is front and which back?
```

Comment:
Seems to be Off - 0, Front - 1, Back - 2

![image](https://user-images.githubusercontent.com/8605238/30329045-d296b4f0-97d9-11e7-91c0-4092d8e3bbfd.png)